scripts/initialise_demo.mjs

```diff
@@ -174,6 +174,50 @@ async function initiateCertificate(hydrogenProducerPort, energyProviderPort) {
   return waitForComplete
 }
 
+// async function checkCarbonIntensityAPI(url, timeout = 2000) {
```

Comment:
Don't comment, just delete

Reply:
With the stuff that has been commented here, I'm not sure that is the correct solution.

Reply:
All cleaned !